.teamcity/_self/projects/WPComTests.kt

```diff
@@ -214,12 +219,23 @@ fun gutenbergBuildType(screenSize: String, buildUuid: String): BuildType {
 	}
 }
 
-fun gutenbergPlaywrightBuildType( targetDevice: String, buildUuid: String ): E2EBuildType {
+fun gutenbergPlaywrightBuildType( targetDevice: String, buildUuid: String, atomic: Boolean = false ): E2EBuildType {
+	var buildId = if (atomic) {
+		"WPComTests_gutenberg_Playwright_atomic_$targetDevice"
+	} else {
+		// buildId should stay the same for non-atomic or TC will not find the entry
+		// and will create a new one. We want to keep all the data/history for it, so
+		// we keep the id as is.
+		"WPComTests_gutenberg_Playwright_$targetDevice"
```

Comment:
I think you can change this string and it _should_ maintain history, because the UUID is the same.

I feel like we can lose the execution history for WPCOM/Gutenberg Playwright build configs, since it's just us that's looking at it at the moment. Let's try changing this string and see if the history gets kept - and we can update the FG page based on the outcome.

Reply:
Sounds good! 👍🏻

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/62037/commits/8998c82ac9f16cf030a3043def66a200b77ae516